src/tests/ftest/cart/util/cart_utils.py

```diff
@@ -276,26 +270,17 @@ def get_yaml_list_elem(param, index):
     def build_cmd(self, env, host, **kwargs):
         """Build a command string."""
 
-        env_ccsa = self.params.get("env", "/run/env_CRT_CTX_SHARE_ADDR/*/")
         test_name = self.params.get("name", "/run/tests/*/")
 
         # Write memcheck result file(s) to $HOME or DAOS_TEST_SHARED_DIR.
-        daos_test_shared_dir = os.getenv('DAOS_TEST_SHARED_DIR',
-                                         os.getenv('HOME'))
-
-        if env_ccsa is None:
-            env_ccsa = ""
-
-        memcheck_xml = r"{}/valgrind.%q\{{PMIX_ID\}}_{}-{}.memcheck".format(daos_test_shared_dir,
-                                                                            test_name, env_ccsa)
-
+        daos_test_shared_dir = os.getenv('DAOS_TEST_SHARED_DIR', os.getenv('HOME'))
+        memcheck_xml = r"{}/valgrind.%q\{{PMIX_ID\}}_{}.memcheck".format(daos_test_shared_dir,
+                                                                            test_name)
```

Comment:
flake8 complains
> E127 continuation line over-indented for visual indent

I'd recommend
```suggestion
        memcheck_xml = r"{}/valgrind.%q\{{PMIX_ID\}}_{}.memcheck".format(
            daos_test_shared_dir,
            test_name)
```

Reply:
will fix